### ISSUE TITLE
why I get a 100% acc on the valid data?

### ISSUE THREAD

**Issue by sosoIrving** [4uiiurz1/keras-arcface#14]

**Comment by vizaiapp** [comment]
Any luck solving this? I haven't been able to reproduce the clusters in the original script yet consistently achieve 100% accuracy on the train and test data for MNIST.

**Comment by ofathy1981** [comment]
> Any luck solving this? I haven't been able to reproduce the clusters in the original script yet consistently achieve 100% accuracy on the train and test data for MNIST.

please can you help me in input dataset to thate model

**Comment by vancuonghoang** [comment]
> _No description provided._

have you done your problem ? let help me pls

**Comment by Reid-T-W** [comment]
I have the same issue, I am consistently getting 100% accuracy on the validation dataset, has anyone been able to solve this?

**Comment by hzfei** [comment]
> 
Any luck solving this?

**Comment by hzfei** [comment]
> I have the same issue, I am consistently getting 100% accuracy on the validation dataset, has anyone been able to solve this?
Any luck solving this?

**Comment by Reid-T-W** [comment]
No, no luck in solving it, but I came across the following article [https://yiwenlai.medium.com/how-to-train-with-arcface-loss-to-improve-model-classification-accuracy-d4035195aeb9](url). It mentions about the 100% accuracy error and how to solve it. Please go through it.

**Comment by hzfei** [comment]
thanks



---Original---
From: ***@***.***&gt;
Date: Wed, Jun 22, 2022 20:10 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [4uiiurz1/keras-arcface] why I get a 100% acc on the valid data?(#14)




 
No, no luck in solving it, but I came across the following article https://yiwenlai.medium.com/how-to-train-with-arcface-loss-to-improve-model-classification-accuracy-d4035195aeb9. It mentions about the 100% accuracy error and how to solve it. Please go through it.

**Comment by hzfei** [comment]
> No, no luck in solving it, but I came across the following article [https://yiwenlai.medium.com/how-to-train-with-arcface-loss-to-improve-model-classification-accuracy-d4035195aeb9](url). It mentions about the 100% accuracy error and how to solve it. Please go through it.

not work

**Comment by ktakanopy** [comment]
Same problem here.

**Comment by hzfei** [comment]
2 ways ，it works&nbsp; &nbsp; 1.use tf 2+&nbsp; 2.
trans speech brain ArcFace to tf version

---Original---
From: "Kevin ***@***.***&gt;
Date: Sun, Aug 28, 2022 06:55 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [4uiiurz1/keras-arcface] why I get a 100% acc on the valid data?(#14)




 
Same problem here.
